.travis.yml

```diff
@@ -1,104 +1,93 @@
 language: php
-
 sudo: false
-
 cache:
-
   directories:
     - $HOME/.composer/cache
+    - $HOME/symfony-bridge/.phpunit
 
 env:
-
   global:
     - SYMFONY_DEPRECATIONS_HELPER="weak_vendors"
     - PHPUNIT_FLAGS="-v"
     - PHPUNIT_ENABLED="true"
     - SYMFONY_PHPUNIT_VERSION="6.5"
-    - PHPCSFIXER_ENABLED="false"
     - COVERALLS_ENABLED="false"
+    - STABILITY=stable
 
 matrix:
-
   fast_finish: true
-
   include:
+    - php: '7.1'
+    - php: '7.2'
+    - php: '7.3'
+      env:
+        - COVERALLS_ENABLED="true"
+        - PHPUNIT_FLAGS="-v --coverage-text --coverage-clover var/build/clover.xml"
 
-    - php: 7.1
-    - php: 7.1
+    # Minimum supported dependencies with the latest and oldest supported PHP versions
+    - php: '7.1'
       env:
-        - PHPUNIT_ENABLED="false"
-        - PHPCSFIXER_ENABLED="true"
-    - php: 7.1
+        - COMPOSER_FLAGS="--prefer-lowest"
+    - php: '7.3'
       env:
-        - SYMFONY_VERSION=4.0.*
         - COMPOSER_FLAGS="--prefer-lowest"
-    - php: 7.2
-    - php: 7.2
-      env: SYMFONY_VERSION=3.4.*
-    - php: 7.2
-      env: SYMFONY_VERSION=4.0.*
-    - php: 7.2
+
+    # Test each supported Symfony version with lowest supported PHP version
+    - php: '7.1'
       env:
-        - DEPENDENCIES="symfony/phpunit-bridge:^4.2"
-        - COMPOSER_UPDATE_FLAGS="--no-dev"
-    - php: 7.2
+        - SYMFONY_VERSION=3.4.*
+    - php: '7.1'
       env:
-        - COVERALLS_ENABLED="true"
-        - PHPUNIT_FLAGS="-v --coverage-text --coverage-clover var/build/clover.xml"
-    - php: 7.2
+        - SYMFONY_VERSION=4.2.*
+    - php: '7.1'
+      env:
+        - SYMFONY_VERSION=4.3.*
+
+    # Test upcoming Symfony versions with lowest supported PHP version and dev dependencies
+    - php: '7.1'
+      env:
+        - STABILITY=dev
+        - SYMFONY_VERSION=4.4.*
+    - php: '7.2'
+      env:
+        - STABILITY=dev
+        - SYMFONY_VERSION=5.0.*
+
+    # Test upcoming PHP versions with dev dependencies
+    - php: '7.4snapshot'
       env:
-        - SYMFONY_VERSION=dev-master
         - STABILITY=dev
+        - COMPOSER_FLAGS="--ignore-platform-reqs"
     - php: nightly
       env:
-        - SYMFONY_VERSION=4.0.*
+        - STABILITY=dev
         - COMPOSER_FLAGS="--ignore-platform-reqs"
-
   allow_failures:
-
-    - env:
-        - COVERALLS_ENABLED="true"
-        - PHPUNIT_FLAGS="-v --coverage-text --coverage-clover var/build/clover.xml"
     - env:
-        - SYMFONY_VERSION=dev-master
         - STABILITY=dev
```

Comment:
according to the [doc](https://docs.travis-ci.com/user/customizing-the-build/#rows-that-are-allowed-to-fail) *all* conditions must match, there is no partial matching. which seems stupid, it would be elegant if this would work as you expect it to.